### DISCUSSION TITLE
How to Fix the Progress Bar

### DISCUSSION THREAD

**Discussion by Shilpa-A12** [ditdot-dev/vue-flow-form#263] [Q&A]
Hi Everyone!

I have used the progress bar = true, to my flow form as in example for - support page 

I have around 15 questions on my project.

I have used the conditional logic in my form as of the support page example.

But the progress bar is not working as expected, It's giving 100% completed for each question, How do I fix this.

Need you suggessions.

Thank you.

**Comment by spinn** [comment]
The progress bar looks at the number of questions in the currently active path. When questions have conditional logic we can't know in advance where the user will go and how many questions are in their path so we calculate up until where we can. What we could do and what we've discussed internally would be to look at the total number of questions, although this also isn't an ideal solution.
